src/transformers/models/regnet/configuration_regnet.py

```diff
@@ -36,6 +36,8 @@ class RegNetConfig(PretrainedConfig):
     documentation from [`PretrainedConfig`] for more information.
 
     Args:
+        image_size (`int`, *optional*, defaults to 224):
+            Size (resolution) of the input images.
```

Comment:
I'd prefer not adding this here. The model works on any image size, adding this in the config might make the user believe it only works for a size of 224.
From what I can see, it's only used in the dummy inputs function, where we can hardcode a size (it's dummy inputs anyway)? Or is TensorFlow going to want always the same size as the dummy inputs?

Reply:
We can hard-code the size here:

https://github.com/ariG23498/transformers/blob/23a20ad2852180096caae98f34f941b41fad55ca/src/transformers/models/regnet/modeling_tf_regnet.py#L424

However, to maintain consistency, I include the `image_size` in the configuration. Would adding a comment (to denote that the model works on any size) to config file help?

Reply:
AFAIK the other models who have this attribute in their config only work on the `image_size` set, so this would confuse users more than anything else, I believe.